phpBB/styles/prosilver/template/index_body.html

```diff
@@ -11,6 +11,8 @@
 <!-- ENDIF -->
 <!-- EVENT index_body_markforums_after -->
 
+Test: {{ macros.Icon('fa', 'phone') }}
```

Comment:
Hmm what’s with the “macros.”

Reply:
As per [this comment](https://github.com/phpbb/phpbb/pull/5545#issuecomment-474015052), it would mean adding a loaded template file to a 'super global'. Which is assigned in the `phpbb/template/twig/extension/macro.php`. Meaning, that all macros defined in the `macros.html` file will be available under `macros.XYZ()`. It is not possible to add them without a 'prefix', eg `{{ Icon() }}`. That would either require still importing them in every file, or creating them as twig functions in php.

Reply:
I see not sure if we want to call it macros then but we can change that later

Reply:
I see not sure if we want to call it macros then but we can change that later